### ISSUE TITLE
Debian piuparts ain't happy about magithub on install process

### ISSUE THREAD

**Issue by mfvescovi** [vermiculus/magithub#200]
[Here](http://debomatic-amd64.debian.net/distribution#unstable/magithub/0.1.5-1/piuparts) is the output of the install/deinstall check made by [piuparts](https://piuparts.debian.org).
While I've found a workaround to Cask issue on compilation (disabling checks on installation), I cannot figure out how to fix this.
Thanks in advance for your time and patience.

**Comment by vermiculus** [comment]
This is almost certainly because I'm using features from magit's HEAD in the stable version of magithub (which admittedly I should've thought about), but it should be resolved once magit tags another release.

Related to #171.

**Comment by vermiculus** [comment]
Closing as a duplicate of #219; note that I plan to wait on releasing 0.2 until this is fixed (which does depend quite a bit on other projects over which I have little influence).
